api/src/main/java/org/pmiops/workbench/conceptset/mapper/ConceptSetMapper.java

```diff
@@ -18,4 +25,47 @@
   @Mapping(target = "etag", source = "version", qualifiedByName = "cdrVersionToEtag")
   @Mapping(target = "concepts", ignore = true)
   ConceptSet dbModelToClient(DbConceptSet source);
+
+  @Mapping(target = "conceptSetId", source = "conceptSet.id")
+  @Mapping(target = "domainEnum", source = "conceptSet.domain")
+  @Mapping(target = "name", source = "conceptSet.name")
+  @Mapping(target = "description", source = "conceptSet.description")
+  @Mapping(
+      target = "creationTime",
+      source = "conceptSet.creationTime",
+      qualifiedByName = "toTimestampCurrentIfNull")
+  @Mapping(
+      target = "lastModifiedTime",
+      source = "conceptSet.lastModifiedTime",
+      qualifiedByName = "toTimestampCurrentIfNull")
+  @Mapping(target = "participantCount", source = "conceptSet.participantCount")
+  @Mapping(target = "domain", ignore = true)
+  @Mapping(target = "survey", ignore = true)
+  @Mapping(target = "creator", ignore = true)
+  @Mapping(target = "surveysEnum", source = "conceptSet.survey")
+  @Mapping(target = "version", source = "conceptSet.etag", qualifiedByName = "etagToCdrVersion")
+  @Mapping(target = "workspaceId", ignore = true)
+  @Mapping(target = "conceptIds", ignore = true)
+  DbConceptSet clientToDbModel(
+      CreateConceptSetRequest source,
+      @Context Long workspaceId,
+      @Context DbUser creator,
+      @Context ConceptBigQueryService conceptBigQueryService);
+
+  @AfterMapping
+  default void updateConceptSetFields(
+      CreateConceptSetRequest source,
+      @Context Long workspaceId,
+      @Context DbUser creator,
+      @MappingTarget DbConceptSet dbConceptSet,
+      @Context ConceptBigQueryService conceptBigQueryService) {
+    dbConceptSet.setWorkspaceId(workspaceId);
+    dbConceptSet.setCreator(creator);
+    dbConceptSet.getConceptIds().addAll(source.getAddedIds());
+    String omopTable = BigQueryTableInfo.getTableName(source.getConceptSet().getDomain());
+    dbConceptSet.setParticipantCount(
+        conceptBigQueryService.getParticipantCountForConcepts(
+            dbConceptSet.getDomainEnum(), omopTable, dbConceptSet.getConceptIds()));
+    dbConceptSet.getConceptIds().addAll(source.getAddedIds());
```

Comment:
Please remove duplicate line of code
```suggestion
    dbConceptSet.getConceptIds().addAll(source.getAddedIds());
    String omopTable = BigQueryTableInfo.getTableName(source.getConceptSet().getDomain());
    dbConceptSet.setParticipantCount(
        conceptBigQueryService.getParticipantCountForConcepts(
            dbConceptSet.getDomainEnum(), omopTable, dbConceptSet.getConceptIds()));
```